kmip/pie/objects.py

```diff
@@ -407,17 +430,21 @@ def validate(self):
     def __repr__(self):
         algorithm = "algorithm={0}".format(self.cryptographic_algorithm)
         length = "length={0}".format(self.cryptographic_length)
-        value = "value={0}".format(self.value)
+        value = "value={0}".format(binascii.hexlify(self.value))
+        format_type = "format_type={0}".format(self.key_format_type)
 
-        return "PublicKey({0}, {1}, {2})".format(algorithm, length, value)
+        return "PublicKey({0}, {1}, {2}, {3})".format(
+            algorithm, length, value, format_type)
 
     def __str__(self):
-        return str(self.value)
+        return str(binascii.hexlify(self.value))
```

Comment:
I went with `binascii.hexlify` here because I just wanted a clear, numeric string representation of the bytes without any strange character displays. If keys are displayed or logged (rarely if ever, I know), would displaying in PEM format be appropriate?